addons/mail/views/mail_message_views.xml

```diff
@@ -147,7 +147,7 @@
                     <field name="name"/>
                     <templates>
                         <t t-name="kanban-box">
-                            <div class="oe_kanban_global_area oe_kanban_global_click o_kanban_attachment">
+                            <div t-attf-class="'oe_kanban_global_area o_kanban_attachment' + #{context.prevent_click ? '' : 'oe_kanban_global_click'}">
```

Comment:
- prefer `{{ }}` over `#{}`
- missing whitespace, should be evaluated as `... o_kanban_attachment oe_kanban_global_click` instead of `... o_kanban_attachmentoe_kanban_global_click` (notice missing whitespace between classnames)

Reply:
Done  :+1: